pkg/syncclient/sync_client.go

```diff
@@ -86,9 +88,20 @@ func (s *SyncerClient) connect(cxt context.Context) error {
 	log.Info("Starting Typha client")
 	var err error
 	logCxt := s.logCxt.WithField("address", s.addr)
+
+	cert, err := tls.LoadX509KeyPair("../certs/client.pem", "../certs/client.key")
+	if err != nil {
+		log.WithError(err).Error("Failed to load certificate and key")
+	}
+	tlsConfig := tls.Config{Certificates: []tls.Certificate{cert}, InsecureSkipVerify: true}
```

Comment:
Assuming this is just WiP but `InsecureSkipVerify` feels like an option we shouldn't be turning on :-)